Signal/src/ViewControllers/BlockListViewController.m

```diff
@@ -137,18 +137,7 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
 - (NSString *)displayNameForIndexPath:(NSIndexPath *)indexPath
 {
     NSString *phoneNumber = _blockedPhoneNumbers[(NSUInteger)indexPath.item];
-    PhoneNumber *parsedPhoneNumber = [PhoneNumber tryParsePhoneNumberFromUserSpecifiedText:phoneNumber];
-
-    // Try to parse and present the phone number in E164.
-    // It should already be in E164, so this should always work.
-    // If an invalid or unparsable phone number is already in the block list,
-    // present it as-is.
-    NSString *displayName = (parsedPhoneNumber ? parsedPhoneNumber.toE164 : phoneNumber);
-    Contact *contact = self.contactMap[displayName];
-    if (contact && [contact fullName].length > 0) {
-        displayName = [contact fullName];
-    }
-
+    NSString *displayName = [_contactsManager displayNameForPhoneIdentifier:phoneNumber];
```

Comment:
Yup, I'm still all for using `[_contactsManager displayNameForPhooneIdentifier:]`.

Here's what I'm talking about from line 146:
```
NSString *displayName = (parsedPhoneNumber ? parsedPhoneNumber.toE164 : phoneNumber);
```

Previous behavior was that if the number was not parseable to e164, to use the raw `phoneNumber`. Now in that case, I think we'll get `nil`.

Like I said, probably doesn't matter since anyone that can be usefully blocked will have an e164 parseable phone number, but I wanted to highlight that the logic was lost in case you were depending on it.